docs/development.md

```diff
@@ -5,13 +5,7 @@
 Create a GCP environment.
 
 ```sh
-docker build ./install -t substratus-installer && \
-
-docker run -it \
-  -v $HOME/.kube:/root/.kube \
-  -e PROJECT=$(gcloud config get project) \
-  -e TOKEN=$(gcloud auth print-access-token) \
-  substratus-installer gcp-up.sh
+make gcp_installer_up
```

Comment:
Perhaps `make install CLOUD=gcp DISABLE_CONTROLLER=true` and move the current `make install` to be `make install-crds`.

Reply:
I'll take one of the two suggestions here and make `DISABLE_CONTROLLER` a flag. Let's leave the `CLOUD` argument until we need it and it does something meaningful 👍